src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -2244,6 +2244,57 @@ public void WriteBase64String(bool formatted, bool skipValidation)
             }
         }
 
+        [Theory]
+        [InlineData(true, true)]
+        [InlineData(true, false)]
+        [InlineData(false, true)]
+        [InlineData(false, false)]
+        public void WriteBase64StringEscapeLengthBigger(bool formatted, bool skipValidation)
```

Comment:
This test doesn't seem to involve JsonDocument, JsonElement, or JsonProperty, and so probably does not belong in this change.

Reply:
Yeah it is not related to JsonDocument or JsonElement test coverage, it is beginning of JsonWriter coverage and added here as it is still related to code coverage and to avoid future file conflicts. But anyways i can remove it if you prefer that

Reply:
Your change title is `Code coverage JsonDocument, JsonElement`.  Nothing should be in the change that is not related to the declared purpose. Assuming there're more code coverage improvements on the way for the writer, this should be held back for those changes.

Reply:
ok, will exclude